lib/i18n.rb

```diff
@@ -1,6 +1,6 @@
 def setup_rails_i18n
   inject_into_class 'app/controllers/application_controller.rb', 'ApplicationController' do
-    "\tinclude Localization\n"
+    "  include Localization\n"
```

Comment:
Why the change here 🤔 Was there any issue with `\t`? Also we keep `\n` so it seems rather inconsistent

Reply:
@olivierobert I changed those because according to `rubocop` tabs are invalid. Hence the spaces.

Reply:
Noted